.ci.yaml

```diff
@@ -4012,7 +4012,7 @@ targets:
     properties:
       tags: >
         ["devicelab", "ios", "mac"]
-      task_name: spell_check_test_ios
+      task_name: spell_check_test
```

Comment:
`spell_check_test_ios` is the task to run. So task name could be different from the name of target (excluding the platform).

Reply:
I see. In TESTOWNERs this is the related task though: 
`/dev/devicelab/bin/tasks/spell_check_test.dart @camsim99 @flutter/android` so should it not be just spell_check_test? Otherwise it does not map.

Reply:
![image](https://github.com/flutter/flutter/assets/32242716/ea4a03bb-16b6-4880-9482-b3fd91ba456a)